### ISSUE TITLE
Automate K selection as per manuscript

### ISSUE THREAD

**Issue by funnell** [friend1ws/pmsignature#2]
Hello,

It would be very useful to have a function that implements the K selection strategy outlined in the pmsignature manuscript. This would save everyone who wants to use the same selection method the effort of implementing it themselves :)

**Comment by friend1ws** [comment]
Hi, thanks for the interest in pmsignature.
Yes, automated selection of the K would be very helpful.
I will consider to implement this in the future.
Plus, giving the summarized results of several K jointly may be also helpful,
so that people can check which K gives the most intuitively suitable result...

**Comment by funnell** [comment]
I agree. Along this line, how would I do this using the available package functions:

> For each of 100 such bootstrap samples, we re-fitted the model, using parameters obtained
> for the original data as initial points

Or do you recommend re-fitting parameters without using the original parameters?

**Comment by friend1ws** [comment]
There is a function called "bootPMSignature".
I think this function is basically OK, but not tested hard compared to other basic functions.
We would like to prepare the this function with more improved document soon.

**Comment by alvinwt** [comment]
Hi, 

I have a question about the function getSignaturesForMultipleK. Can it be used to generate signatures over a range of K? Is there a way to run it to approximate the optimal K value?I have problem visualizing the output as well as trying to determine the optimal K. 

There is a small bug which returns that object IsBackGround not found when running the function. You might want to include BG argument in the function as well. It would be good if the function returns a list of values for each K, with the output as well.

**Comment by friend1ws** [comment]
Hi, 

The function getSignaturesForMultipleK is deprecated..
The current version (0.2) now changed the visPMSignature function so that ggplot2 figure is generated, and this is not compatible with the procedure generating multiple figures in getSignaturesForMultipleK.
I'm not sure whether I create similar function with getSignaturesForMultipleK  function soon. I'm very sorry for inconvenience..
